### PR TITLE
SRCNN_tensorflow

### PR DESCRIPTION
**Pull request by shvmshri**

### CODE REVIEW

File: super_resolution/SRCNN_tensorflow/README.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+SRCNN involves  predicting  HR images from LR images making use of a CNN architecture.
+
+#### Model Architecture
+
+SRCNN network is actually not much deep. It uses a standard CNN architecture. We take HR images, then downsampl them to images of low-spatial resolution. 
+
+*	The low-resolution input is then first upscaled to the desired size using inter-cubic interpolation before inputting it to the SRCNN     network.
+*	We take patches of a particular size from the image and pass them through the network.
+*	Then, standard convolution layers using an activation function are applied which predict the HR version of that patch.
+     ![alt text](https://github.com/shvmshri/model-zoo/blob/master/super_resolution/SRCNN_tensorflow/assets/srcnn.png)
```
Comment (nirmal-suthar) [inline super_resolution/SRCNN_tensorflow/README.md]: Use ./assets/* everywhere

**Comment by nirmal-suthar** [comment]
Also remove dataset folder and move all images you need in readme from dataset to assets folder

File: super_resolution/SRCNN_tensorflow/main.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+       X_test = X_test.reshape(X_test.shape[1],X_test.shape[2])
+       Y_pred = Y_pred.reshape(Y_pred.shape[1],Y_pred.shape[2])
+   
+       cv2.imshow(X_test)
+       cv2.imshow(Y_test)
+       cv2.imshow(Y_pred)
+     
+
+
+
```
Comment (nirmal-suthar) [inline super_resolution/SRCNN_tensorflow/main.py]: remove stray lines

**Review by nirmal-suthar — APPROVED** [review]

**Comment by shvmshri** [comment]
> Also remove dataset folder and move all images you need in readme from dataset to assets folder

dataset folder contains images on which i have trained my model and also testing ,result images..These are not required in readme

**Comment by nirmal-suthar** [comment]
Datasets are generally bulky, so we don't include raw dataset in this repo. Instead, you could add the download link for a dataset in the readme file.

File: super_resolution/SRCNN_tensorflow/main.py (hunk 2 above)
Comment (shvmshri) [reply super_resolution/SRCNN_tensorflow/main.py]: done

File: super_resolution/SRCNN_tensorflow/README.md (hunk 1 above)
Comment (shvmshri) [reply super_resolution/SRCNN_tensorflow/README.md]: done

**Comment by shvmshri** [comment]
> Datasets are generally bulky, so we don't include raw dataset in this repo. Instead, you could add the download link for a dataset in the readme file.

i customized this dataset according to my needs ,hence there is no official link.Do i need to create it then?

**Comment by nirmal-suthar** [comment]
> i customized this dataset according to my needs ,hence there is no official link.Do i need to create it then?

Ohh, then simply provide the link to the original dataset in readme, and write it there, that you used subset of that dataset.

**Pull request merged**